.github/workflows/check.yml

```
@@ -17,44 +17,52 @@ jobs:
       run:
         shell: bash
     strategy:
+      fail-fast: true
       matrix:
         system:
           - ubuntu-latest
           - macos-latest
           - windows-latest
+        version:
+          - '1.0.1'
+          - '2.18.1'
```

```suggestion
          - 1.0.1
          - 2.18.1
```

Quotes not needed for `major.minor.patch` nor for `major`; only for `major.minor` because it can be mistaken with a float.

---

style wise the base branch had these quoted (previously in the `with:` block), I saw no reason to change it.

---

👌🏼